app.py

```diff
@@ -261,7 +274,7 @@ def handle_postback():
     payment_amount = citybase_data["data"]["total_amount"]
     citybase_id = citybase_data["data"]["id"]
     app.logger.info(
-        f"Payment status: {payment_status}, invoice number: {knack_invoice}"
+        f"{citybase_id} - Payment status: {payment_status}, invoice number: {knack_invoice}"
```

Comment:
Awesome learning for me here -- seeing how you're sticking the ID of the event family into the logs for easier sorting & filtering is really great. Awesome pattern - thank you!